### ISSUE TITLE
Normal and Specular maps

### ISSUE THREAD

**Issue by ekvasir** [NCDyson/StudioCCS#2]
not really an issue but just wanted to share that i found that some models in vol 4 have Normal and Specular maps.
The image i have attached shows the textures and shaders for Vegalta

https://steamuserimages-a.akamaihd.net/ugc/922543230728661230/096FD47CD3968ABC6179FE9CDB979A31C738D83D/

**Comment by NCDyson** [comment]
yeah, I think there's a material parameter somewhere. haven't had a chance to check on that.
thinking about opening issues for all the stuff I need to work on, just so that I can remember it all.

**Comment by ekvasir** [comment]
Probably not a bad idea also that way if anyone comes across anything of interest they can let you know
